File: src/G2_py/PYA202.py
n=eval(input())

# 判斷它是3或5的倍數
if (n%3 == 0) & (n%5 == 0):
    print("{:} is a multiple of 3 and 5.".format(n))
elif(n%5 == 0):
    print("{:} is a multiple of 5.".format(n))
elif(n%3 == 0):
    print("{:} is a multiple of 3.".format(n))
else:
    print("{:} is not a multiple of 3 or 5.".format(n))

File: src/G2_rust/PYA202.rs
use std::io;

fn main(){
    let mut input: String = String::new();
    io::stdin().read_line(&mut input).unwrap();
    let n: i32 = input.trim().parse().unwrap();
    let result_false: i32 = 0;

    if n%3 == result_false && n%5 == result_false {
        print!("{} is a multiple of 3 and 5.",n);
    }else if n%5 == result_false {
        print!("{} is a multiple of 5.",n);
    }else if n%3 == result_false {
        print!("{} is a multiple of 3.",n);
    }else{
        print!("{} is not a multiple of 3 or 5.",n);
    }
}